doc/docs/deploy-manage/deploy/environment-variables.md

```diff
@@ -0,0 +1,166 @@
+
+You can configure environment variables as you typically do
+in a Linux environment. For example, to configure a `PACH_JOB_ID`
+in bash, run:
+
+```bash
+$ export PACH_JOB_ID=<ID>
+```
+
+You can access all the variables in the Pachyderm manifest that
+is generated when you run `pachctl deploy` with the --dry-run`
+flag.
+
+!!! note "See also"
+    [Deploy Pachyderm](../../../getting_started/local_installation/#deploy-pachyderm)
+
+## `pachd` Environment Variables
+
+You can find the list of `pachd` environment variables in the
+`pachd` manifest by running the following command:
+
+```bash
+$ kubectl get deploy pachd -o yaml
+```
+
+The following tables list all the `pachd`
+environment variables.
+
+**Global Configuration**
+
+| Environment Variable   | Default Value     | Description |
+| ---------------------- | ----------------- | ----------- |
+| `ETCD_SERVICE_HOST`    | N/A               | The host on which the etcd service runs. |
+| `ETCD_SERVICE_PORT`    | N/A               | The etcd port number.                    |
+| `PPS_WORKER_GRPC_PORT` | `80`              | The GRPs port number.                    |
+| `PORT`                 | `650`             | The `pachd` port number. |
+| `PPROF_PORT`            | `651`             | The port number for the `pprof` tool.     |
+| `HTTP_PORT`             | `652`             | The HTTP port number.   |
+| `PEER_PORT`             | `653`             | The port for pachd-to-pachd communication. |
+| `PPS_ETCD_PREFIX`        | `pachyderm_pps`   | ???  |
+| `NAMESPACE`            | `deafult`         | The namespace in which Pachyderm is deployed. |
+| `PACH_ROOT`          | `/pach`           | ??? |
+
+**pachd Configuration**
+
+| Environment Variable   | Default Value     | Description |
+| ---------------------- | ----------------- | ----------- |
+| `NUM_SHARDS`           | `32`              | The max number of `pachd` pods that can run in a single cluster. |
+| `STORAGE_BACKEND`      | `"`               | The storage backend defined for the Pachyderm cluster. By default, local storage is configured. |
+| `STORAGE_HOST_PATH`    | `"`               | The host path to storage.                    |
+| `ETCD_PREFIX`          | `"`               | A prefix for the etcd cluster.  |???
+| `PFS_ETCD_PREFIX`      | `pachyderm_pfs`   | The ???  - Should we expose this in the docs
+| `PACHYDERM_AUTH_ETCD_PREFIX` | `pachyderm_auth` |   ???? Should we expose this in the docs?
+| `PACHYDERM_ENTERPRISE_ETCD_PREFIX` | `pachyderm_enterprise` | ???? should we expose in the docs
+| `KUBERNETES_PORT_443_TCP_ADDR` | N/A   | A Kubernetes TCP address. |
+| `METRICS`              | `true`         | Defines whether anonymous Pachyderm metrics are being collected or not. |
+| `INIT`                 | `false`           | ???
+| `BLOCK_CACHE_BYTES`    | `1G`              | The size of the block cache in `pachd`.  |
+| `PFS_CACHE_SIZE`       | `0`               | The size of the PFS cache for PFS objects.  |
```

Comment:
We could expose this but I'm not sure if we want to, just because its meaning is strange. It's the number of hashtrees (PFS metadata) that we cache in memory. The default value of the env var is `0`, as noted, but internally we convert that a different default of `8`. I think we might not want to expose this in docs just because I can't imagine a user wanting to adjust this on their own. I think it's only exposed because of PFS experiments we've done.